src/overlays/effects/ovl_Effect_Ss_Extra/z_eff_ss_extra.c

```diff
@@ -34,8 +34,8 @@ u32 EffectSsExtra_Init(PlayState* play, u32 index, EffectSs* this, void* initPar
     s32 objectSlot;
 
     objectSlot = Object_GetSlot(&play->objectCtx, OBJECT_YABUSAME_POINT);
-    if ((objectSlot > OBJECT_SLOT_NONE) && (Object_IsLoaded(&play->objectCtx, objectSlot))) {
-        void* segBackup = gSegments[6];
+    if ((objectSlot >= 0) && (Object_IsLoaded(&play->objectCtx, objectSlot))) {
```

Comment:
Just a bad merge. Could you bring back `OBJECT_SLOT_NONE` and remove the extra brackets around `Object_IsLoaded`